[2025-03-12T03:52:13.806+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T03:52:13.828+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.bigquery_external_table_task manual__2025-03-12T03:51:57.199522+00:00 [queued]>
[2025-03-12T03:52:13.838+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.bigquery_external_table_task manual__2025-03-12T03:51:57.199522+00:00 [queued]>
[2025-03-12T03:52:13.839+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T03:52:13.852+0000] {taskinstance.py:2890} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2025-03-12 03:51:57.199522+00:00
[2025-03-12T03:52:13.859+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=481) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-12T03:52:13.861+0000] {standard_task_runner.py:72} INFO - Started process 483 to run task
[2025-03-12T03:52:13.862+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'bigquery_external_table_task', 'manual__2025-03-12T03:51:57.199522+00:00', '--job-id', '294', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp6_hfq_9u']
[2025-03-12T03:52:13.863+0000] {standard_task_runner.py:105} INFO - Job 294: Subtask bigquery_external_table_task
[2025-03-12T03:52:13.904+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_gcs_dag.bigquery_external_table_task manual__2025-03-12T03:51:57.199522+00:00 [running]> on host b0577ed9eb82
[2025-03-12T03:52:13.991+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='bigquery_external_table_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T03:51:57.199522+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T03:51:57.199522+00:00'
[2025-03-12T03:52:13.994+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-12T03:52:13.995+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-12T03:52:13.995+0000] {logging_mixin.py:190} INFO - Current task name:bigquery_external_table_task state:running start_date:2025-03-12 03:52:13.829570+00:00
[2025-03-12T03:52:13.996+0000] {logging_mixin.py:190} INFO - Dag name:data_ingestion_gcs_dag and current dag run status:running
[2025-03-12T03:52:13.996+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T03:52:13.998+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-12T03:52:13.999+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-03-12T03:52:14.000+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-03-12T03:52:15.488+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T03:52:15.489+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=bigquery_external_table_task, run_id=manual__2025-03-12T03:51:57.199522+00:00, execution_date=20250312T035157, start_date=20250312T035213, end_date=20250312T035215
[2025-03-12T03:52:15.513+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-12T03:52:15.514+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-12T03:52:15.514+0000] {logging_mixin.py:190} INFO - Dag name:data_ingestion_gcs_dag queued_at:2025-03-12 03:51:57.223333+00:00
[2025-03-12T03:52:15.515+0000] {logging_mixin.py:190} INFO - Task hostname:b0577ed9eb82 operator:BigQueryCreateExternalTableOperator
[2025-03-12T03:52:15.562+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T03:52:15.577+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-12T03:52:15.579+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
